.eslintrc

```diff
@@ -249,11 +253,11 @@ rules:
   sort-imports: [0]
   sort-keys: [0]
   space-before-blocks: [2, always]
-  space-before-function-paren: [2, never]
+  space-before-function-paren: [0, never]
```

Comment:
hehe ok... I'm not a mind reader 😉... I'll take care of it later today. I have to run now! 😸